tests/ui/alloc-error/alloc-error-backtrace.rs

```diff
@@ -0,0 +1,49 @@
+//@ run-pass
```

Comment:
I think this test doesn't work on `thumb7em-unknown-none`. Can you add it to `ignored-tests.toml`? I don't really think this test is relevant to the target anyways since it seems to be testing backtraces which aren't a thing on them?